From 140f7e295b60a20a26c51a8fd007de1b9f3761b5 Mon Sep 17 00:00:00 2001
From: Ken Brown <kbrown@cornell.edu>
Date: Wed, 28 Nov 2018 00:43:21 +0000
Subject: [PATCH 2/3] fix InputData constructor

Initialize InputData::frontendSections.
---
 src/inputdata.h | 1 +
 1 file changed, 1 insertion(+)

diff --git a/src/inputdata.h b/src/inputdata.h
index ba263a59..44b06b52 100644
--- a/src/inputdata.h
+++ b/src/inputdata.h
@@ -172,6 +172,7 @@ struct InputData
 	InputData( const HostLang *hostLang, struct colm_sections *frontendSections, struct colm_sections *rlhcSections )
 	: 
 		FsmGbl(hostLang),
+		frontendSections(frontendSections),
 		rlhcSections(rlhcSections),
 		inputFileName(0),
 		outputFileName(0),
-- 
2.17.0

