--- origsrc/ragel-7.0.0.11/src/Makefile.am	2018-05-18 12:42:37.000000000 -0400
+++ src/ragel-7.0.0.11/src/Makefile.am	2018-11-14 17:35:09.132763500 -0500
@@ -35,6 +35,8 @@ data_DATA = ragel.lm
 #
 libfsm_la_CPPFLAGS = -I$(top_srcdir)/aapl
 
+libfsm_la_LDFLAGS = -no-undefined
+
 dist_libfsm_la_SOURCES = \
 	buffer.h codegen.h \
 	actloop.h actexp.h \
@@ -61,36 +63,40 @@ nodist_libfsm_la_SOURCES = \
 #
 libragel_la_CPPFLAGS = -I$(top_srcdir)/aapl -DBINDIR='"@bindir@"'
 
+libragel_la_LDFLAGS = -no-undefined
+
+libragel_la_LIBADD = libfsm.la $(COLM_LD)
+
 dist_libragel_la_SOURCES = \
 	parsedata.h parsetree.h inputdata.h pcheck.h reducer.h rlscan.h load.h \
 	parsetree.cc parsedata.cc inputdata.cc load.cc reducer.cc
 
-#
-# ragel program.
-#
-ragel_CPPFLAGS = -I$(top_srcdir)/aapl
-
-dist_ragel_SOURCES = \
-	main.cc
-
-nodist_ragel_SOURCES = \
+nodist_libragel_la_SOURCES = \
 	parse.c rlreduce.cc
 
-ragel_LDADD = libragel.la libfsm.la $(COLM_LD)
-ragel_DEPENDENCIES = libragel.la libfsm.la $(COLM_LIBDEP)
-
 BUILT_SOURCES = \
 	version.h \
 	parse.c rlreduce.cc
 
 if WITH_RAGEL_KELBT
-nodist_ragel_SOURCES += \
+nodist_libragel_la_SOURCES += \
 	rlscan.cc rlparse.cc rlparse.h 
 
 BUILT_SOURCES += \
 	rlscan.cc rlparse.cc rlparse.h 
 endif
 
+#
+# ragel program.
+#
+ragel_CPPFLAGS = -I$(top_srcdir)/aapl
+
+dist_ragel_SOURCES = \
+	main.cc
+
+ragel_LDADD = libragel.la libfsm.la $(COLM_LD)
+ragel_DEPENDENCIES = libragel.la libfsm.la $(COLM_LIBDEP)
+
 version.h: Makefile
 	echo '#define VERSION "$(PACKAGE_VERSION)"' > version.h
 	echo '#define PUBDATE "$(PUBDATE)"' >> version.h
